lib/fluent/command/fluentd.rb

```diff
@@ -81,6 +81,23 @@
   opts[:log_path] = s
 }
 
+op.on('--log-rotate-age AGE', 'generations to keep rotated log files') {|age|
+  rotate_ages = %w(daily weekly monthly)
+  if rotate_ages.include?(age)
+    opts[:log_rotate_age] = age
+  else
+    begin
+      opts[:log_rotate_age] = Integer(age)
```

Comment:
Are there any reason to use `Integer()` instead of `to_i`?
IMO `to_i` is widely used and normal for most ruby programmer.